pkg/oc/clusteradd/components/default-imagestreams/create_secret.go

```diff
@@ -75,7 +75,8 @@ func (opt *installReadyDockerConfigSecret) Install(dockerClient dockerhelper.Int
 
 	dockerConfigJsonBytes, err := ioutil.ReadFile(path.Join(home, ".docker", "config.json"))
 	if err != nil {
-		return fmt.Errorf("Error reading $HOME/.docker/config.json: %v", err)
+		glog.Warningf("Error reading $HOME/.docker/config.json: %v, imagestream import credentials will not be setup", err)
+		return nil
```

Comment:
why are you returning?

Reply:
because if we can't read the config.json we can't create the import secret from the config.json.

Reply:
You shouldn't be returning here.

Reply:
I see, this is installing the secret.